### ISSUE TITLE
Curator 4 Apt Packages

### ISSUE THREAD

**Issue by DustinChaloupka** [elastic/curator#929]
Hi!

We have been installing version `4.2.6` of the `elasticsearch-curator` apt package for awhile, but now it seems that it is not an available version with `4.3.0` having been released on a new VM. The [deb package](http://packages.elastic.co/curator/4/debian/pool/main/e/elasticsearch-curator/elasticsearch-curator_4.2.6_amd64.deb) for `4.2.6` is still available, but the [packages](http://packages.elastic.co/curator/4/debian/dists/stable/main/binary-amd64/Packages) does not list the `4.2.6` version.

Is this intentional to remove the older versions from the repository? It's not too big of a deal for us to upgrade to the newest stable version, but just wanted to make sure.

**Comment by untergeek** [comment]
Thanks for bringing it to my attention.

We had an AWS host where we did our package staging and repository management get deleted 😮 by accident.

I had to rebuild from scratch, and though the S3 repository was intact, the scripts we used were gone.  I ended up using reprepro to rebuild the APT repo, and that's the behavior it has.  I'm going to investigate using another repo manager like aptly to see if it will preserve the older versions in the Packages file.
